File: sections/samples/get.js
const request = require('request');
const email = 'johndoe@email.com';
const password = 'secret';

request.get(
    'https://app.paymoapp.com/api/projects',
    {
        auth: {
            user: email,
            pass: password
        },
        headers: {
            'Accept': 'application/json'
        }
    },
    function (error, response, body) {
        if (!error) {
            // List project names
            JSON.parse(body).projects.forEach(function (project) {
                console.log(project.name);
            });
        } else {
            console.log(error);
        }
    }
);

File: sections/samples/get.py
import requests

email = 'johndoe@email.com'
password = 'secret'

headers = {'Accept': 'application/json'}
r = requests.get('https://app.paymoapp.com/api/projects', headers=headers, auth=(email, password))

# List project names
for project in r.json()['projects']:
    print project['name']
